atomix/cluster/src/main/java/io/atomix/cluster/messaging/impl/NettyMessagingService.java

```diff
@@ -725,7 +725,9 @@ private CompletableFuture<Channel> bootstrapClient(final Address address) {
                   if (!onConnect.isSuccess()) {
                     future.completeExceptionally(
                         new ConnectException(
-                            String.format("Failed to connect channel for address %s", address)));
+                            String.format(
+                                "Failed to connect channel for address %s (resolved: %s) : %s",
+                                address, address.address(), onConnect.cause())));
```

Comment:
Lets follow our error guidelines :) https://github.com/camunda/zeebe/wiki/Error-Guidelines

```suggestion
                            String.format(
                                "Expected to create channel to %s (resolved: %s), but failed with : %s",
                                address, address.address(), onConnect.cause())));
```

Reply:
I find it unnecessarily verbose. When we say "Failed to connect channel" It is already clear that it is expected to connect channel. Right? Besides the error guidelines doesn't require to use the exact same format.

Reply:
I prefer a consistent format, even if it is a bit more verbose it is always clear what is expected and what happened with additional context. As far as I know, we agreed on that a while ago. If you disagree, feel free to bring it up in a team discussion.

Reply:
BTW go ahead to merge it anyway, nothing what should block you (I already accepted)